package.json

```diff
@@ -6,9 +6,12 @@
     "cross-spawn": "^7.0.3",
     "husky": "^7.0.4",
     "i18next-scanner": "^3.1.0",
-    "prettier": "^2.6.2"
+    "prettier": "^2.6.2",
+    "eslint": "^7.32.0",
+    "lint-staged": "^12.3.8"
```

Comment:
interesting to see we use this from root package but they comes from underlying packages :)